indiserver/Constants.hpp

```diff
@@ -0,0 +1,42 @@
+/* INDI Server for protocol version 1.7.
+ * Copyright (C) 2007 Elwood C. Downey <ecdowney@clearskyinstitute.com>
+                 2013 Jasem Mutlaq <mutlaqja@ikarustech.com>
+                 2022 Ludovic Pollet
+    This library is free software; you can redistribute it and/or
+    modify it under the terms of the GNU Lesser General Public
+    License as published by the Free Software Foundation; either
+    version 2.1 of the License, or (at your option) any later version.
+
+    This library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+    Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public
+    License along with this library; if not, write to the Free Software
+    Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+*/
+#pragma once
+
+#define INDIPORT      7624    /* default TCP/IP port to listen */
+#define MAXSBUF       512
+#define MAXRBUF       49152 /* max read buffering here */
+#define MAXWSIZ       49152 /* max bytes/write */
+#define SHORTMSGSIZ   2048  /* buf size for most messages */
+#define DEFMAXQSIZ    128   /* default max q behind, MB */
+#define DEFMAXSSIZ    5     /* default max stream behind, MB */
+#define DEFMAXRESTART 10    /* default max restarts */
+#define MAXFD_PER_MESSAGE 16 /* No more than 16 buffer attached to a message */
+#ifdef OSX_EMBEDED_MODE
+#define LOGNAME  "/Users/%s/Library/Logs/indiserver.log"
+#define FIFONAME "/tmp/indiserverFIFO"
+#endif
+
+extern int verbose;
+extern unsigned int maxstreamsiz;
+extern unsigned int maxqsiz;
+extern char *ldir;
+extern int maxrestarts;
+
+class Fifo;
+extern Fifo * fifo;
```

Comment:
These externs were just added as a means of sharing them temporarily between the files. With some light refactoring of the locations they're in use, these can all be removed.